plutus-core/cost-model/CostModelGeneration.md

```diff
@@ -2,17 +2,18 @@
 
 Generating a cost model for CPU time involves a number of steps.
 
-*  Run `cabal bench plutus-core:cost-model-budgeting-bench` on the
+*  Run `cabal run plutus-core:cost-model-budgeting-bench -- --csv <file>` on the
   reference machine.  This will run Criterion benchmarks for the built-in
   functions and will take many hours.  Each function is run many times with a
   selection of inputs of different sizes.  The benchmarks for the builtins are
   small, executing single Plutus Core terms on the CEK machine.
 
 *  The results of the benchmarks (execution versus sizes of inputs) are
-   stored in a CSV file called `benching.csv` in `plutus-core/cost-model/data`.
-   There's a checked-in copy [here](./data/benching.csv).
+   stored in CSV format in the file named in the `--csv` option, which is mandatory.
 
-* Run `cabal bench plutus-core:update-cost-model`.  This runs some R code in
+* Change directory to `plutus-core/cost-model/data/` and run
+  `cabal bench plutus-core:generate-cost-model -b <file>`, where `<file>` is the
```

Comment:
```suggestion
  `cabal run plutus-core:generate-cost-model -- -b <file>`, where `<file>` is the
```

Reply:
Also, what does `-b` stand for? Would it confuse you if it was instead again `--csv`?

Reply:
Yes, why not?  I thought about making it `-i` for input like uplc, but `--csv` seems sensible (although the other --csv is forced on us by Criterion).